File: javaproject/src/p2023_07_14/For04.java
package p2023_07_14;

public class For04 {
public static void main(String[] args) {
	
	// 1~100 까지 홀수, 짝수의 합을 구하는 프로그래 ㅁ작성
	// 단, for문 1개와 if~else문으로 작성
	
	int odd = 0, even = 0;
	
	for (int i = 1; i<=100; i++) {
		if (i%2==1) {
			odd += i;
		} else {
			even +=i;
		}
	}
	System.out.println("홀수 : " + odd + " 짝수 : " + even);
}
}

File: Python_1127/2023_11_29/control/for04.py
# 반복문 : for문

# for  변수  in  range() :
#    반복 실행할 문장

# 1 ~ 100 까지 홀수의 합과 짝수의 합을 구하는 프로그램 작성

# 홀수의 합 구하기
odd = 0
for i in range(1, 101, 2):
    odd += i
print('1~100 까지 홀수의 합:', odd)

# 짝수의 합 구하기
even = 0
for i in range(0, 101, 2):
    even += i
print('1~100 까지 짝수의 합:', even);

